python/chronos/src/bigdl/chronos/forecaster/prophet_forecaster.py

```diff
@@ -100,6 +102,10 @@ def predict(self, horizon=1, freq="D", ds_data=None):
                the frequency can be anything from the pandas list of frequency strings here:
                https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#timeseries-offset-aliases
         :param ds_data: a dataframe that has 1 column 'ds' indicating date.
+
+        :return: A pandas DataFrame of length horizon,
+                 including "trend" and "seasonality" and inference values, etc.
+                 where the last column is the inference value.
```

Comment:
We may state the last column's name. It's "yhat"?

Reply:
You are right, I will change this.